kobo/apps/subsequences/constants.py

```diff
@@ -1,5 +1,8 @@
+from typing import Any, List
```

Comment:
`List` is deprecated. Use `from __future__ import annotations` and use lowercase type everywhere instead.
See https://docs.python.org/3/library/typing.html#deprecated-aliases

Reply:
Ok